pkg/apis/integreatly/v1alpha1/installation_types.go

```diff
@@ -30,6 +31,15 @@ var (
 	ProductFuse                ProductName = "fuse"
 	Product3Scale              ProductName = "3scale"
 	ProductNexus               ProductName = "nexus"
+
+	VersionAMQStreams          ProductVersion = "1.1"
+	VersionAMQOnline           ProductVersion = "1.1"
+	VersionSolutionExplorer    ProductVersion = "2.10"
+	VersionRHSSO               ProductVersion = "7.3"
+	VersionCodeReadyWorkspaces ProductVersion = "1.2"
+	VersionFuse                ProductVersion = "1.7"
+	Version3Scale              ProductVersion = "2.6"
+	VersionNexus               ProductVersion = "3.16"
```

Comment:
This seems a pretty gross way to do this. I know at least some operators have a version field in the status block. How would we update these versions with Delorean?

Reply:
I've done this as much as is reasonable now.

Reply:
Ok lets add a comment to why we have these hard coded vs the others otherwise it could be confusing when others read the code here.

Reply:
done!